chartfx-math/src/main/java/de/gsi/math/DataSetMath.java

```diff
@@ -850,45 +849,45 @@ public static DataSet mathFunction(final DataSet function1, final DataSet functi
             switch (op) {
             case ADD:
                 newY = Y1 + Y2;
-                nEYN = Math.hypot(eyn1, eyn2);
-                nEYP = Math.hypot(eyp1, eyp2);
+                nEYN = java.lang.Math.hypot(eyn1, eyn2);
+                nEYP = java.lang.Math.hypot(eyp1, eyp2);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.hypot' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)